src/main/java/de/uhd/ifi/se/decision/management/jira/model/KnowledgeGraph.java

```diff
@@ -36,27 +42,73 @@
 	 *            of the Jira project.
 	 * @return either a new or already existing KnowledgeGraph instance.
 	 * 
-	 * @see KnowledgeGraphImpl
+	 * @see KnowledgeGraph
 	 */
-	static KnowledgeGraph getOrCreate(String projectKey) {
+	public static KnowledgeGraph getOrCreate(String projectKey) {
 		if (projectKey == null || projectKey.isBlank()) {
 			return null;
 		}
 		if (instances.containsKey(projectKey)) {
 			return instances.get(projectKey);
 		}
-		KnowledgeGraph knowledgeGraph = new KnowledgeGraphImpl(projectKey);
+		KnowledgeGraph knowledgeGraph = new KnowledgeGraph(projectKey);
 		instances.put(projectKey, knowledgeGraph);
 		return knowledgeGraph;
 	}
 
-	static KnowledgeGraph getOrCreate(DecisionKnowledgeProject project) {
+	public static KnowledgeGraph getOrCreate(DecisionKnowledgeProject project) {
 		if (project == null) {
 			return null;
 		}
 		return getOrCreate(project.getProjectKey());
 	}
 
+	private static final Logger LOGGER = LoggerFactory.getLogger(KnowledgeGraph.class);
+
+	private static final long serialVersionUID = 1L;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5381317)